lib/peer-list.ts

```diff
@@ -14,31 +14,31 @@ export default class PeerList extends EventEmitter {
     }
 
     public addPeer(name: string, res: IPeerResponse, req: IPeerRequest) {
-        this.emit(SignalEvent.PrePeerAdd, name);
+        this.emit("addPeer:pre", name);
```

Comment:
aww no more enum? :(

Reply:
I noticed that Publisher uses the [old string version](https://github.com/bengreenier/webrtc-signal-http-publisher) version for hooking into signaler. I thought it might be confusing to have to import the enum to use it, so I reverted. If this is not the right strategy please let me know!

Reply:
maybe we can swap our usage of "EventEmitter" to https://www.npmjs.com/package/strict-event-emitter-types and then use strings everywhere? that way the transpiler can help us with autocomplete still, but we'll use string literals.

Reply:
Oh neat! Will update to that

Reply:
Updated!